trigger/task.go

```diff
@@ -457,7 +456,6 @@ func (t *taskTriggers) taskFirstFailureInBuild(sub *event.Subscription) (*notifi
 	if t.task.DisplayOnly {
```

Comment:
I know I put this here, but is this check necessary?
(and in taskFirstFailureInVersion)

Reply:
I believe so. For these first failure, it's often that someone wants to know as soon as we know the task will fail, which is this point. And if we go that route (which we are here), we have to alert on execution tasks, and alerting for display tasks would be duplicative. I think this also answers your other questions, but let me know if it doesn't

Reply:
but is it necessary to check? Even without checking, we won't alert for the display task because it's not the first failure.

Reply:
Yes because this is a cheaper way to no-op, since letting it through would make us do a query before no-oping